codegen/src/main/kotlin/com/android/designcompose/codegen/BuilderProcessor.kt

```diff
@@ -958,7 +958,7 @@ class BuilderProcessor(private val codeGenerator: CodeGenerator, val logger: KSP
             out.appendText("                customizations = customizations,\n")
             out.appendText("                modifier = modifier,\n")
             out.appendText(
-                "                serverParams = DocumentServerParams(queries, nodeCustomizations(), ignoredImages()),\n"
+                "                serverParams = DocumentServerParams(queries, ignoredImages()),\n"
```

Comment:
It's a bit more work, but are you planning on removing the code that generates the nodeCustomizations() function?

Reply:
Good call, removing that now.